examples/alignment/pede/job.json

```diff
@@ -0,0 +1,17 @@
+{
+    "input_files":  {
+        "/full/path/to/step/1/output/events_1_mille.bin" : "events_1_mille.bin",
```

Comment:
We should get 3 files to use for this example hosted at JLab so we can put the web link to the file here.

Reply:
I agree it would be nice to have publicly accessible files for testing purposes instead of dummy paths. I would be okay either with http access via the test dir at JLab or site-specific files (JLab would be fine with me).